src/providers/PushProvider.ts

```diff
@@ -13,7 +13,7 @@ export function scheduleNotification(options: {title: string; body: string; sche
   let notificationId = 0;
   try {
     if (storedNotificationId !== null && storedNotificationId !== '') {
-      notificationId = parseInt(JSON.parse(storedNotificationId));
+      notificationId = parseInt(JSON.parse(storedNotificationId)) + 1;
```

Comment:
Do you also save the notificationId in localStorage?

Reply:
Yea i store it under the try catch, was just an increment issue i think